karma.config.cjs

```diff
@@ -23,7 +23,9 @@ module.exports = async config => {
       { pattern: 'test/**/*.test.js', type: 'module', watched: false },
       { pattern: 'test/assets/**', watched: false, included: false }
     ],
-
+    exclude: [
```

Comment:
why is this removed?

Reply:
There was a test I guess thats why @nilshah98  had to remove it.

Reply:
> First sdk-utils/request , here we have added support for fully qualified urls and also for https server.

> Hence, to support that while testing, created a local instance of https server and wrote tests accordingly, which passed.

> The current issue is, that sdk-utils also run their tests on browsers, where it tries to create this https server and fails.

> As a workaround I have added test/request.test.js in the exclude parameter in karma.config.js

Reply:
I think it would be due to the SSL error it's breaking, lets try fixing this. Excluding it here doesn't seem clean. lets sync on this.

Reply:
For now add a comment here and request.test.js why we're skipping it.

Reply:
Done 👍